.gitignore

```diff
@@ -1,3 +1,7 @@
 target
 Cargo.lock
 /.cargo/config
+.project
+.settings
+*~
+*.bk
```

Comment:
Seems like you should add all of that to a global ignore file anyway or you are going to need to add it in all projects all the time.